package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fastify/deepmerge",
```

Comment:
could you add the ci.yml?
https://github.com/fastify/fastify-url-data/blob/master/.github/workflows/ci.yml

Reply:
It already exists, just added the lint:true flag in the ci.yml to make it visible